doc/source/software_using_gdal.rst

```diff
@@ -104,6 +104,7 @@ Software using GDAL
 - `StarSpan  <https://github.com/Ecotrust/starspan>`_  raster/vector analysis.
 - `TacitView  <https://www.insitu.com/information-delivery/information-processing/tacitview>`_  An imagery visualization and exploitation package for military intelligence.
 - `TatukGIS  <http://www.tatukgis.com>`_  Desktop GIS mapping and data editing application.
+- `Team Awareness Kit <https://tak.gov>`_  Suite of georeferenced imagery and situational awareness tools. Developed for military planning and execution for U.S. Special Operations Command Operators. TAK is also used by Fire Fighters, Emergency Management, First Responders and Law Enforcement.
```

Comment:
Can you make the description more concise similar to other entries ?

Reply:
Modified to be more concise. Thanks